lib/fuel-auth.js

```diff
@@ -20,19 +20,26 @@ module.exports = class FuelAuth {
 			if (typeof options.clientId !== 'string' || typeof options.clientSecret !== 'string') {
 				throw new Error('clientId or clientSecret must be strings');
 			}
+
+			if(options.apiVersion === 2 && !options.tenant){
+				throw new Error('Version 2 APIs require that you specify a tenant');
+			}
 		} else {
 			throw new Error('options are required. see readme.');
 		}
 
 		// set required values
 		this.accessToken = options.accessToken;
-		this.authUrl = options.authUrl || 'https://auth.exacttargetapis.com/v1/requestToken';
+		this.authUrl = options.authUrl || (options.version === 2 ? `https://${options.tenant}.auth.marketingcloudapis.com/v2/token` : 'https://auth.exacttargetapis.com/v1/requestToken');
```

Comment:
Why are we asking `options` to have `version` and `apiVersion`? Can we just have one? `version` is only being used for this default behavior statement, but everywhere else `apiVersion` is being used.

1. Keep using `version` but just change the assignment to `options.apiVersion = options.version`
2. Just make `apiVersion` the parameter to check in this default case.

Reply:
Hi there :) 
version is loading the version of the package, whereas apiVersion is which oauth version to use. I didnt want to use an existing version variable as it may cause breaks in other features.